src/scenekit.cs

```diff
@@ -68,9 +68,13 @@
 #if MONOMAC
 using AppKit;
 
+using EAGLContext = Foundation.NSObject;
```

Comment:
using EAGLContext = Foundation.NSObject; is defined 3 times here. Can we combine them to something like
 
``` 
#if MONOMAC || WATCH || __MACCATALYST__
 using EAGLContext = System.Object;
 #endif
```